src/main/java/org/minions/devfund/sergio/Squarelotron.java

```diff
@@ -0,0 +1,208 @@
+package org.minions.devfund.sergio;
+
+import java.util.Arrays;
+
+/**
+ * Main methods for game Squarelotron.
+ *
+ * @author Sergio Navarro
+ * @since 4/20/2018
+ */
+public class Squarelotron {
+
+    /**
+     * Internal Array.
+     */
+    private int[][] squarelotron;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename field "squarelotron" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1700)

Reply:
Please rename the variable 
Take a look of 
https://sonarcloud.io/coding_rules?open=squid%3AS1700&rule_key=squid%3AS1700